examples/synopsysctl/securityContext.json

```diff
@@ -1,61 +1,70 @@
 {
     "blackduck-authentication": {
-        "fsGroup": 1000,
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "fsGroup": 1234,
+        "runAsUser": 1234,
+        "runAsGroup": 1234
+    },
+    "blackduck-postgres": {
+        "fsGroup": 26,
+        "runAsUser": 26,
+        "runAsGroup": 26
+    },
+    "blackduck-init": {
+        "runAsUser": 1234,
+        "runAsGroup": 1234
     },
     "blackduck-binaryscanner": {
-        "fsGroup": 1000,
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "fsGroup": 1234,
+        "runAsUser": 1234,
+        "runAsGroup": 1234
     },
     "blackduck-cfssl": {
-        "fsGroup": 1000,
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "fsGroup": 1234,
+        "runAsUser": 1234,
+        "runAsGroup": 1234
     },
     "blackduck-documentation": {
-        "fsGroup": 1000,
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "fsGroup": 1234,
+        "runAsUser": 1234,
+        "runAsGroup": 1234
     },
     "blackduck-jobrunner": {
-        "fsGroup": 1000,
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "fsGroup": 1234,
+        "runAsUser": 1234,
+        "runAsGroup": 1234
     },
     "blackduck-rabbitmq": {
-        "fsGroup": 1000,
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "fsGroup": 1234,
+        "runAsUser": 1234,
+        "runAsGroup": 1234
     },
     "blackduck-registration": {
-        "fsGroup": 1000,
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "fsGroup": 1234,
+        "runAsUser": 1234,
+        "runAsGroup": 1234
     },
     "blackduck-scan": {
-        "fsGroup": 1000,
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "fsGroup": 1234,
+        "runAsUser": 1234,
+        "runAsGroup": 1234
     },
     "blackduck-uploadcache": {
-        "fsGroup": 1000,
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "fsGroup": 1234,
+        "runAsUser": 1234,
+        "runAsGroup": 1234
     },
     "blackduck-webapp": {
-        "fsGroup": 1000,
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "fsGroup": 1234,
+        "runAsUser": 1234,
+        "runAsGroup": 1234
     },
     "blackduck-logstash": {
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "runAsUser": 1234,
+        "runAsGroup": 1234
     },
     "blackduck-nginx": {
-        "fsGroup": 1000,
-        "runAsUser": 1000,
-        "runAsGroup": 1000
+        "fsGroup": 1234,
```

Comment:
Can we keep the numbers same as before? i.e 1000

Reply:
Done